src/components/Notification.tsx

```diff
@@ -0,0 +1,71 @@
+import React, { ComponentProps, FC, ReactNode, useCallback, useRef, useState } from 'react'
+import { TransitionGroup } from 'react-transition-group'
+import { css } from '../../styled-system/css'
+import { token } from '../../styled-system/tokens'
+import FadeTransition from './FadeTransition'
+import PopupBase from './PopupBase'
+
+/** An alert component in which you can customize what is rendered. Used for Alerts + Tips. */
+const Notification: FC<
+  {
+    renderedIcon?: ReactNode
+    value: ReactNode | null
+    /* Specify a key to force the component to re-render and thus recalculate useSwipeToDismissProps when the alert changes. Otherwise the alert gets stuck off screen in the dismiss state. */
+    transitionKey: string | number
```

Comment:
This solution feels a bit hacky, but should be acceptable if we at least ensure unique keys are being passed. See (4).